styles/tool-bar.less

```diff
@@ -38,6 +38,7 @@
     }
   }
 
+  // Set the margin between the buttons (that are not at the end)
```

Comment:
Comment seems a little redundant to me, the LESS is pretty clear I would say :thinking: In fact, the LESS is removing the left/top margin and not setting any margins

Reply:
You are right. My background isn't in web development, so it wasn't clear to me at first glance. Since the sentence was a little complicated, I thought it is worth mentioning the function. 

What comment do you suggest here instead?

Reply:
In that case it's fine if the comment is there, changing "set" to "configure" would help but it's not necessary.